index.html

```diff
@@ -387,6 +371,49 @@ <h2>
         applied.
       </aside>
     </section>
+    <section>
+      <h2>
+        Calculating an accessible name
+      </h2>
+      <p>
+        The steps for <dfn data-lt="calculate an accessible name">calculating an
+        accessible name</dfn> are given by the following algorithm. The
+        algorithm takes an [=image resource=] |input| and an optional
+        [=Response=] |image| which is the result of [=fetching an image
+        resource=]. It returns a [=string=].
```

Comment:
This is doing a bit too much in the preamble of the algorithm here, as "fetching and image" could fail. The image resource should optionally fetched beforehand, rather than here (and then the calculated accessible named can be derived whenever it's needed).